compiler-core/src/io/memory.rs

```diff
@@ -236,6 +323,30 @@ impl FileSystemReader for InMemoryFileSystem {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum InMemoryFileRepr {
```

Comment:
What's a repr? We don't abbreviate in this codebase so a full name should be used.

Reply:
Okay! I meant "representation" here so I'll probably just change to that, though I admit this might not be the best name either way; I'm open to other ideas.